WalletWasabi.Gui/WalletWasabi.Gui.csproj

```diff
@@ -67,5 +67,14 @@
     <RuntimeHostConfigurationOption Include="System.Globalization.Invariant" Value="true" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
unsure where does it come from. Will remove later.

Reply:
Yes please, it's some Avalonia issue, we keep constantly removing this accidental csproj trashing.

Reply:
I think you mean visual studio issue ;)